portfolio/index.html

```diff
@@ -0,0 +1,137 @@
+<link href="https://fonts.googleapis.com/css?family=Syncopate" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300" rel="stylesheet">
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+
+<div class="container-fluid">
+
+    <ul class = "nav nav-pills">
+      <li class="nav-item pull-right">
+        <a class="nav-link" href="#contact">Contact</a>
+      </li>
+      <li class="nav-item pull-right">
+        <a class="nav-link" href="#portfolios">Portfolio</a>
+      </li>
+      <li class="nav-item pull-right">
+        <a class="nav-link" href="#about">About Me</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="#pageOne">Arti M</a>
+      </li>
+    </ul>
+
+    <div class="pageOne" id="pageOne">
+      <h1 class="text-center">Arti M</h1>
+      <h2 class="text-center">Jill of all trades. </br> Master of some.</h2>
+    </div>
+
+    <div class="pageTwo" id="about">
+      <h1 class="text-left">About Me</h1>
+      <h3 class="text-left">
+        <p>
+          I have a Masters Degree in Computing, I'm a Certified Scrum
+          Practitioner.
+        </p>
+        <p>
+          I've been part of the tech industry for a little over a decade and
+          donned various hats - QA, PM, Agile Coach, Product Lead. I have
+          been dabbling in code for many many years.
+        </p>
+        <p>
+          I have worked in 8 cities, 5 countries, 3 continents.
+        </p>
+        <p>
+          I am incredibly lucky to be working in a field that is relevant pretty
+          much everywhere in the world and that it is a field that I love.
+        </p>
+      </h3>
+
+  </div>
+
+  <div class="pageThree" id="portfolios">
+    <h1 class="text-center">Portfolio</h1>
+    <h3 class="text-center">A bunch of things I've worked on and placeholders
+        for others that I will work on shortly.
+        <br>
+      </h3>
+    <div class="container-fluid">
+      <div class="container row">
```

Comment:
Here is the only place where you want to use a `container` div but I don;t think you should nest a `container` inside  `container-fluid` or set `row` as a `container`. So this is what you want:
```html
<div class="container-fluid">
  <div class="row">
    ...
  </div>
</div.
```